GCMessengerSDKSample/app/src/main/java/com/genesys/cloud/messenger/sample/MainActivity.kt

```diff
@@ -235,6 +236,10 @@ class MainActivity : AppCompatActivity(), ChatEventListener {
     private fun createChat(account: AccountInfo, chatStartError: (() -> Unit)? = null) {
         waitingVisibility(true)
 
+        if (account is MessengerAccount && viewModel.isAuthenticated){
+            account.setAuthenticationInfo(viewModel.authCode.value!!, viewModel.redirectUri, viewModel.codeVerifier)
```

Comment:
Can we avoid using the !! operator?

Reply:
I would not. We checked isAuthenticated() right before this line.